### ISSUE TITLE
`id` not included in response to `list`

### ISSUE THREAD

**Issue by jml** [warner/magic-wormhole#259]
`server-protocol.md` says

> Some client->server messages (`list`, `allocate`, `claim`, `release`, `close`, `ping`) provoke a direct response by the server: for these, `id` is copied into the response. 

But if you look at a transcript of messages you see this:

```
>>>
{u'id': u'ce03', u'type': u'list'}
>>>

<<<
{u'id': u'ce03', u'server_tx': 1508679418.989837, u'type': u'ack'}
<<<

<<<
{u'nameplates': [], u'server_tx': 1508679418.989998, u'type': u'nameplates'}
<<<
```

Note that the `nameplates` message has no `id` field.

**Comment by jml** [comment]
Just noticed that it's similar for all "RPCs".

A quick scan through https://github.com/warner/magic-wormhole/blob/master/src/wormhole/server/rendezvous_websocket.py shows that none of the `handle_` methods add an id to the outgoing message.

**Comment by warner** [comment]
Good point.. that should be fixed. `id` is only for debugging (really it's only for the timing-diagram performance-analysis tools), but I'm all for consistency.

It looks like I included `id=` in subscription-like messages, which really only means the `message` events that are sent if you'd previously done a `open`. And in the `ack` that gets sent in response to every command. That's all we needed for the timing tools.

Lemme think about whether it's better to add `id` to every immediate response, or to change the docs to stop claiming that this happens. I'm leaning towards the former.
